vendor/cran.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'cran', title: 'CRAN/METACRAN' });
+module.exports = t;
```

Comment:
I see that often too. Though, when I do that, I always forget to export it, and the errors that result are really opaque. So in modules like this which only export one thing, I export as early as possible. Seems particularly valuable here to keep all the boilerplate above the dev's code.